tests/dataset/test_anndata.py

```diff
@@ -99,6 +100,41 @@ def test_transfer_anndata_setup():
     assert adata2.obs["_scvi_labels"][0] == 0
 
 
+def test_data_format():
+    # if data was dense np array, check after setup_anndata, data is C_CONTIGUOUS
+    adata = synthetic_iid(run_setup_anndata=False)
+
+    old_X = adata.X
+    old_pro = adata.obsm["protein_expression"]
+    old_obs = adata.obs
+    adata.X = np.asfortranarray(old_X)
+    adata.obsm["protein_expression"] = np.asfortranarray(old_pro)
+    assert adata.X.flags["C_CONTIGUOUS"] is False
+    assert adata.obsm["protein_expression"].flags["C_CONTIGUOUS"] is False
+
+    setup_anndata(adata, protein_expression_obsm_key="protein_expression")
+    assert adata.X.flags["C_CONTIGUOUS"] is True
```

Comment:
let's make sure using get from registry gives the same results

Reply:
I think I had noticed a weird issue where you set adata.X of a registered object and get from registry still references the old object

Reply:
I would still add one line to the test to make sure.